pkg/models/secrets_client.go

```diff
@@ -40,13 +40,6 @@ func NewSecretsClient(namespace string) (*SecretsClient, error) {
 		return nil, err
 	}
 
-	if namespace == "" {
-		namespace, _, err = kubeConfig.Namespace()
-		if err != nil {
-			return nil, err
-		}
-	}
-
```

Comment:
This needs to stay here, otherwise when `ksec` is used as a standalone binary it will default to returning all secrets in all namespaces. The desired behavior is for it to default to whatever is set in kubeconfig.

We should also probably have a unit test fails if this block of code is removed.

Reply:
makes sense. I've updated the mock client + added a separate unit test